docs/20-Sortable-admin-list.md

```diff
@@ -0,0 +1,272 @@
+# Admin sortable list
+
+To sort an admin list by drag&drop
+
+![sortable-admin-list.png](img/sortable-admin-list.png)
+
+## How it works
+
+Specific templates are used by your admin to add a class to HTML table to init javascript drag&drop:
+- sonata/src/Ressources/views/CRUD/list_outer_rows_sortable_list.html.twig
+- sonata/src/Ressources/views/CRUD/list_sortable.html.twig
+
+When you drop the row of your list, an ajax call send all positions of your items list to a new route of your admin to save all positions
+
+## Installation
+
+You must install some javascript libraries to your project :
+ - "@shopify/draggable": "^1.0.0-beta.4"
+ - "superagent": "3.8.2"
+ 
+You must include css sonata/src/Ressources/public/css/sortableAdminList.css to your project
+ 
+You must call the sonata/src/Ressources/public/js/backoffice.js to init drag&drop if a sortable list is display
+
+You must declare a new service
+```
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+
+        <!--SortAction-->
+        <service id="admin.sort_action" class="SortAction\SortAction">
+            <argument type="service" id="doctrine" />
+        </service>
+
+    </services>
+</container>
+```
+
+## How to implement drag&drop to your admin
+
+To use drag&drop to your entity admin, you must update your code like below
+
+### Position field
```

Comment:
Can't the extension provide this through DI? (Maybe optionally) It might ease out the install process :)

Reply:
ok, I add DI for service `admin.sort_action`